tests/hardhat/Rewards.ts

```diff
@@ -222,8 +222,12 @@ describe("Rewards: Tests", async function () {
     await rewardsDistributor.initialize(comptrollerProxy.address, xvs.address);
     await rewardsDistributor2.initialize(comptrollerProxy.address, mockDAI.address);
 
-    await comptrollerProxy.addRewardsDistributor(rewardsDistributor.address);
-    await comptrollerProxy.addRewardsDistributor(rewardsDistributor2.address);
+    await expect(comptrollerProxy.addRewardsDistributor(rewardsDistributor.address))
```

Comment:
I wouldn't add `expect` statements in the `before` function. I would create a specific test to check the event is emitted

Reply:
Done